config/env/default.js

```diff
@@ -431,5 +431,14 @@ module.exports = {
 	/*
 	 * Configurations for External Services
 	 */
-	external: {}
+	external: {},
+
+	/**
+	 * Configurations for configuring Prometheus metrics exporter
+	 */
+	metrics: {
+		enabled: false,
+		prefix: 'starter',
+		collectDefaultMetrics: true
```

Comment:
@gu - are there any performance / memory considerations for collecting metrics by default? What if a fork doesn't use them?

Reply:
@brianghig I haven't done any direct performance comparisons with default metrics disabled. If they are disabled, then the output of the metrics route would essentially be an empty object, which prometheus will be able to handle.

Hoping that the implementation for our team will give us a better idea on performance impact